.github/workflows/root-workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Root Workflow
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  automated-fixes: 
+    uses: ./.github/workflows/autofixes.yml
+    secrets: inherit
+
+  update-tailwind-bundle: 
+    uses: ./.github/workflows/update-tailwind-bundle.yml
+    secrets: inherit
+
+  run-tests:
+    needs: automated-fixes
```

Comment:
We want to prevent our tests from running not only when automated-fixes fails but ALSO when it succeeds but puts up a commit
I think the easiest way would be to fail the autofixes workflow if a commit was made? Not sure exactly how you'd do that, but I think you can use the output from the add-and-commit action
https://github.com/EndBug/add-and-commit#outputs

Reply:
I think that is taken care of by the `concurrency` above!  Whenever another commit is pushed to a PR, it will cancel everything (automated fixes, unit tests, playwright tests, tw bundling) and only run the new checks.